### PR TITLE
Merge Github Action in master branch

### PR DESCRIPTION
**Pull request by Poulinhoo**

### CODE REVIEW

File: 10/action.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: lint JS
+author: PrestaShop
+description: 'An easy GA to lint your JS projet without dev dependencies'
+branding:
+  icon: 'check'
+  color: 'blue'
+inputs:
+  cmd:
+    description: 'Add some config for vuejs project'
```
Comment (zalexki) [inline 10/action.yml]: why vuejs ? please reword (same below with the path description)

File: 10/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+set -e
+
+if [ ! -z "$2" ]
+then
+  echo "Changing dir to $2"
+  cd $2
+fi
+
+echo "Using $1 to install dependencies"
+$1 install
```
Comment (zalexki) [inline 10/entrypoint.sh:11]: you said above `JS projet without dev dependencies` yet here you install dev deps no ?

File: 12/action.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: lint JS
+author: PrestaShop
+description: 'An easy GA to lint your JS projet without dev dependencies'
+branding:
+  icon: 'check'
+  color: 'blue'
+inputs:
+  cmd:
+    description: 'Add some config for vuejs project'
```
Comment (zalexki) [inline 12/action.yml]: same

**Review by zalexki — CHANGES_REQUESTED** [review]

File: 10/action.yml (hunk 1 above)
Comment (Poulinhoo) [reply 10/action.yml]: the linter is often used when we do vuejs that's why I write vuejs

Comment (zalexki) [reply 10/action.yml]: then you understand why it's wrong ?

File: 10/entrypoint.sh (hunk 2 above)
Comment (Poulinhoo) [reply 10/entrypoint.sh:11]: No, I install all the dependencies because I need a vue-cli to be able to launch the linter.

Comment (zalexki) [reply 10/entrypoint.sh:11]: ye ok then you need to fix the description

Comment (Poulinhoo) [reply 10/entrypoint.sh:11]: yes already fixed

File: 10/action.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+name: lint JS
+author: PrestaShop
+description: 'An easy GA to lint your JS projet'
+branding:
+  icon: 'check'
+  color: 'blue'
+inputs:
+  cmd:
+    description: 'add to your package manager js like npm or yarn'
```
Comment (zalexki) [inline 10/action.yml]: please reword this sentence doesn't mean anything. 
maybe you should find another param name i don't see the link between cmd and the package manager 🤔

File: 10/action.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    required: true
+  path:
+    description: 'Add path to your js project'
```
Comment (zalexki) [inline 10/action.yml]:
```suggestion
    description: 'Specify a path to your js project'
```

File: latest/action.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+name: lint JS
+author: PrestaShop
+description: 'An easy GA to lint your JS projet'
+branding:
+  icon: 'check'
+  color: 'blue'
+inputs:
+  cmd:
+    description: 'add to your package manager js like npm or yarn'
```
Comment (zalexki) [inline latest/action.yml]: same as above

File: 12/action.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
+name: lint JS
+author: PrestaShop
+description: 'An easy GA to lint your JS projet'
+branding:
+  icon: 'check'
+  color: 'blue'
+inputs:
+  cmd:
+    description: 'add to your package manager js like npm or yarn'
```
Comment (zalexki) [inline 12/action.yml]: same comment

**Review by zalexki — CHANGES_REQUESTED** [review]

File: 10/action.yml (hunk 4 above)
Comment (Poulinhoo) [reply 10/action.yml]: I think the therme package manager is not appropriate, but the cmd link is for the run command if it is npm or yarn

Comment (Poulinhoo) [reply 10/action.yml]: i think can i change with 

> Specify if you using npm or yarn

**Review by zalexki — APPROVED** [review]

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**